configurecompiler.cmake

```diff
@@ -593,7 +590,7 @@ if (WIN32)
   # enable control-flow-guard support for native components for non-Arm64 builds
   # Added using variables instead of add_compile_options to let individual projects override it
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /guard:cf")
-  set(CMAKE_C_FLAGS "${CMAKE_CXX_FLAGS} /guard:cf")
```

Comment:
is this intentional?

Reply:
Yes. We shouldn’t be setting the C flags with the C++ flags. I think this is a typo that hasn’t been an issue so far because of how early this line is hit in the CMake script.